helper/student.js

```diff
@@ -0,0 +1,62 @@
+const Student  = require('../models/student');
+const callbacks = {};
+
+callbacks.getStudentList = ( req, res ) => {
+      // console.log(req.query);
+      Student.find(req.query)
+      .then( s => {
+            res.json(s);
+      })
+      .catch( err => {
+            console.log(err);
+            res.status(404).json({message: err.message});
```

Comment:
Instead of console log, can we do something like logger.log?
Also, Before putting logger err, can we also add one logger statement saying some text about the expected error, so that it's easier to see & debug.

Valid for all occurrences where we're logging/catching errors.